server/parsec/components/memory/datamodel.py

```diff
@@ -106,6 +107,9 @@ class MemoryDatamodel:
     unverified_emails: dict[EmailAddress, tuple[EmailValidationToken, DateTime]] = field(
         default_factory=dict
     )
+    accounts_in_deletion: dict[EmailAddress, tuple[AccountDeletionToken, DateTime]] = field(
```

Comment:
"in deletion" sounds like we are currently deleting. I cannot find a better name though, maybe something like:
```suggestion
    accounts_deletion_requested: dict[EmailAddress, tuple[AccountDeletionToken, DateTime]] = field(
```

Reply:
done